configs/rec/svtr/svtr_tiny.yaml

```diff
@@ -57,9 +72,9 @@ metric:
   print_flag: False
 
 loss:
-  name: CTCLoss 
-  pred_seq_len: 25
-  max_label_len: *max_text_len  # this value should be smaller than pre_seq_len
+  name: CTCLoss
+  pred_seq_len: 25 # 100 / 4
```

Comment:
How can we confirm the optimal value for `pred_seq_len`?

Reply:
it is based on the network output, cannot be tuned unless we modify the head